pom.xml

```
@@ -11,10 +11,12 @@
     </parent>
     <groupId>com.vaadin</groupId>
     <artifactId>vaadin-archetypes</artifactId>
-    <version>13.0.2</version>
+    <version>13.0-SNAPSHOT</version>
     <packaging>pom</packaging>
 
-
+    <properties>
+        <vaadin.version>${project.version}</vaadin.version>
```

```suggestion
        <vaadin.version>13.0.2</vaadin.version>
```

Project version is not the same as vaadin version used by outputted project. If we build an archetype with SNAPSHOT vaadin version, we should have snapshot repository declared.

---

updated the `vaadin.version`; 
removed the snapshot repository, as it is not needed if we use a release platform version